src/components/Header.tsx

```diff
@@ -0,0 +1,22 @@
+import Icon from './Icon';
+import Username from './Username';
+import ProfilePicture from './ProfilePicture';
+import ellipsis from '../assets/ellipsis.svg';
+import { classes } from '../style/comments.st.css';
+
+interface IPostOwner {
+    name: string;
```

Comment:
this should be userName or something a bit for descriptive as to where this appears